### PR TITLE
Dex 86043 fix parsing errors

### PR DESCRIPTION
**Pull request by evansmith**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- *This work is behind a feature toggle (flipper): NO*
- Changed the logging in the retries_exhausted method to pass the error message from the first msg argument to prevent an error occurring during logging.
- VA Benefits and Claims

## Related issue(s)

[- *Link to ticket created in va.gov-team repo OR screenshot of Jira ticket if your team uses Jira*](https://github.com/department-of-veterans-affairs/va.gov-team/issues/86043)
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*

## Testing done

- [ ] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by evansmith** [comment]
Here are screenshots of the new logs showing the messages successfully: 
Screenshot 2024-08-14 at 10 30 19 AM
Screenshot 2024-08-14 at 10 33 19 AM

File: app/sidekiq/bgs/submit_form674_job.rb [hunk 1]
```diff
@@ -13,11 +13,11 @@ class Invalid674Claim < StandardError; end
 
     sidekiq_options retry: 14
 
-    sidekiq_retries_exhausted do |msg, error|
+    sidekiq_retries_exhausted do |msg, _error|
       user_uuid, icn, saved_claim_id, encrypted_vet_info, encrypted_user_struct_hash = msg['args']
       vet_info = JSON.parse(KmsEncrypted::Box.new.decrypt(encrypted_vet_info))
-      Rails.logger.error('BGS::SubmitForm674Job failed, retries exhausted...',
-                         { user_uuid:, saved_claim_id:, icn:, error: })
+      Rails.logger.error("BGS::SubmitForm674Job failed, retries exhausted! Last error: #{msg['error_message']}",
```
Comment (mjknight50) [inline app/sidekiq/bgs/submit_form674_job.rb:19]: Any chance msg is blank?
Perhaps we should do a check for msg?

File: app/sidekiq/bgs/submit_form686c_job.rb [hunk 2]
```diff
@@ -13,11 +13,11 @@ class Invalid686cClaim < StandardError; end
 
     sidekiq_options retry: 14
 
-    sidekiq_retries_exhausted do |msg, error|
+    sidekiq_retries_exhausted do |msg, _error|
       user_uuid, icn, saved_claim_id, encrypted_vet_info = msg['args']
       vet_info = JSON.parse(KmsEncrypted::Box.new.decrypt(encrypted_vet_info))
-      Rails.logger.error('BGS::SubmitForm686cJob failed, retries exhausted!',
-                         { user_uuid:, saved_claim_id:, icn:, error: })
+      Rails.logger.error("BGS::SubmitForm686cJob failed, retries exhausted! Last error: #{msg['error_message']}",
```
Comment (mjknight50) [inline app/sidekiq/bgs/submit_form686c_job.rb:19]: Same as above

**Review by mjknight50 — APPROVED** [review]
GTG

**Review by ericboehs — APPROVED** [review]

**Pull request merged**